### DISCUSSION TITLE
ipv6和防火墙

### DISCUSSION THREAD

**Discussion by lj0307222** [immortalwrt/immortalwrt#999] [Q&A]
### Describe the bug

如图，好像屏蔽了ipv6链接
![Screenshot_20230630-211023](https://github.com/immortalwrt/immortalwrt/assets/61595433/098dce2c-0ac8-4498-947d-f91bd455c008)
![Screenshot_20230630-205637](https://github.com/immortalwrt/immortalwrt/assets/61595433/bd974162-d7d6-4226-a4a4-f8962ffd8ef3)

### ImmortalWrt version

23.05

### ImmortalWrt target/subtarget

x86/64

### Device

x86

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by cherinyy** [comment]
测试 IPv6 网络连通性建议前往 [IPv6 测试](https://testipv6.cn/)，ImmortalWrt 的防火墙规则默认没有拦截 IPv6 流量，请自行检查配置是否没有问题。

**Reply by lj0307222** [reply]
![Screenshot_20230630-213342](https://github.com/immortalwrt/immortalwrt/assets/61595433/3b4e8e23-35e7-49d7-9b85-c07b6880aebf)
![Screenshot_20230630-213331](https://github.com/immortalwrt/immortalwrt/assets/61595433/689ab76a-99aa-4e00-8ac9-0f110923138c)
设置和以前的一样，23.05就出问题了

**Comment by cherinyy** [comment]
有一种可能性是在接口配置中没有勾选“默认路由”。
